src/index.html

```diff
@@ -11,7 +11,182 @@
+          <input
+            type="checkbox"
+            name="agreement"
+            id="agreement"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form-fieldset">
+        <legend>Registration</legend>
+
+        <div class="form-field">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="5"
+            maxlength="15"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form-fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+          <input
+            type="radio"
+            name="cats"
+            value="yes"
+            id="loveCats"
+          >
+          <label for="loveCats">Yes</label>
+
+          <input
+            type="radio"
+            name="cats"
+            value="no"
+            id="dontLoveCats"
+          >
+          <label for="dontLoveCats">No</label>
+        </div>
+
+        <div class="form-field">
+          <label for="color">What is your favorite color?</label>
+          <input
+            type="color"
+            name="color"
+            id="color"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="bedTime">What time do you go to bed?</label>
+          <input
+            type="time"
+            name="bedTime"
+            id="bedTime"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="carBrand">What are your favorite brand of cars?</label>
+          <select
+            name="carBrand"
+            id="carBrand"
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </div>
+
+        <div class="form-field">
+          <label for="rateOurWork">How do you rate our work?</label>
+          <input
+            type="range"
+            name="rateOurWork"
+            id="rateOurWork"
+            value="0"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form-fieldset">
+        <legend>Additional info:</legend>
+
+        <div class="form-field">
+          <label for="comments">Comments:</label>
+          <textarea
+            name="comments"
+            id="comments"
+          >
+          </textarea>
+        </div>
+
+        <div class="form-field">
+          <label for="recommend">Would you recommend us?</label>
+          <select
+            name="recommend"
+            id="recommend"
+          >
```

Comment:
If you have less then 3 attributes, you can fit them in one line
```suggestion
          <select name="recommend" id="recommend">
```